app/models/plip.rb

```diff
@@ -11,13 +11,28 @@ class Plip
   #   @return [Phase]
   attr_accessor :phase
 
+  # @!attribute [rw] presentation
+  #   @return [String]
+  attr_accessor :presentation
+ 
+  # @!attribute [rw] signatures_required
+  #   @return [String]
+  attr_accessor :signatures_required
+ 
+  # @!attribute [rw] call_to_action
+  #   @return [String]
+  attr_accessor :call_to_action
+
   # @param document_url [String]
   # @param content [String]
   # @param phase [Phase]
-  def initialize(document_url:, content:, phase:)
+  def initialize(document_url:, content:, phase:, presentation:, signatures_required:, call_to_action:)
     @document_url = document_url
     @content = content
     @phase = phase
+    @presentation = presentation
```

Comment:
I guess for now it is okay to pass down all these attrs here. Shortly this will change anyway. Later we can passa the petition object, etc.